src/test/orphanage_tests.cpp

```diff
@@ -29,14 +31,16 @@ class TxOrphanageTest : public TxOrphanage
     CTransactionRef RandomOrphan()
     {
         std::map<Wtxid, OrphanTx>::iterator it;
-        it = m_orphans.lower_bound(Wtxid::FromUint256(InsecureRand256()));
+        it = m_orphans.lower_bound(Wtxid::FromUint256(m_rng.rand256()));
         if (it == m_orphans.end())
             it = m_orphans.begin();
         return it->second.tx;
     }
+
+    FastRandomContext& m_rng;
```

Comment:
Data-members should be private by default:
```diff
--- a/src/test/orphanage_tests.cpp
+++ b/src/test/orphanage_tests.cpp
@@ -20,6 +20,8 @@ BOOST_FIXTURE_TEST_SUITE(orphanage_tests, TestingSetup)
 
 class TxOrphanageTest : public TxOrphanage
 {
+    FastRandomContext& m_rng;
+
 public:
     TxOrphanageTest(FastRandomContext& rng) : m_rng{rng} {}
 
@@ -36,8 +38,6 @@ public:
             it = m_orphans.begin();
         return it->second.tx;
     }
-
-    FastRandomContext& m_rng;
 };
 
 static void MakeNewKeyWithFastRandomContext(CKey& key, FastRandomContext& rand_ctx)
```
Same thing with `CPartialMerkleTreeTester`.

Reply:
Defaulting to private means that should something outside the class start needing access to it, that PR would need to expose it, thereby highlighting to reviewers what is actually happening.

I think it's a good principle even if this is test code, it is new in this PR after all. Maybe if you re-touch?